flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/plan/stream/sql/join/JoinTest.xml

```diff
@@ -444,9 +444,8 @@ SELECT * FROM
 LogicalProject(i=[$0], j=[$1], t=[$2], i0=[$3], k=[$4])
 +- LogicalJoin(condition=[AND(=($0, $3), =($0, $4))], joinType=[full])
    :- LogicalTableScan(table=[[MyTable3, source: [TestTableSource(i, j, t)]]])
-   +- LogicalSort(sort0=[$0], sort1=[$1], dir0=[DESC-nulls-last], dir1=[ASC-nulls-first])
-      +- LogicalProject(i=[$0], k=[$1])
-         +- LogicalTableScan(table=[[MyTable4, source: [TestTableSource(i, k)]]])
+   +- LogicalProject(i=[$0], k=[$1])
+      +- LogicalTableScan(table=[[MyTable4, source: [TestTableSource(i, k)]]])
```

Comment:
It seems that Calcite removes `LogicalSort` in the initial RelNode tree. And it makes sense.